### ISSUE TITLE
accessibility issues with the pihole web-ui

### ISSUE THREAD

**Issue by mhussaincov** [pi-hole/AdminLTE#1174]
**In raising this issue, I confirm the following:** `{please fill the checkboxes, e.g: [X]}`

- [] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [] The issue I am reporting can be *replicated*.
- [] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the the source code relevant to this issue?:**
1
`{Replace this with a number from 1 to 10. 1 being not familiar, and 10 being very familiar}`

---
**Expected behaviour:**
I use a screen reader orca on debian linux, the parts of the web-ui that are not accessible are the folowing 
Total queries over last 24 hours
after that is spoken by orca, the graphic part is not accessible,
all that orca the screen reader will tell me is canvus clickable.

nore is the part write after this bit.
Client activity over last 24 hours
the same is  repeated.

`{A detailed description of what you expect to see}`
would it be possible to have a text version of the graphical parts of the webui in text format for people like me who don't have site?
simply a text version of the pieces of information that is reprozented by the graphics?
e.g the urls accessed etc?

**Actual behaviour:**
the parts stated above are not accessible to orca the screen reader.
`{A detailed description and/or screenshots of what you do see}`

**Steps to reproduce:**
1.
launch orca from your linux distribution of choise most distributions have it pre installed.
2.
go to your pihole admin interface and log in.
3.
navigate to
Client activity over last 24 hours
and down arrow once.
you will hear no useful information after that 

this happens in the firefox browser and chromium, one

`{Detailed steps of how we can reproduce this}`

**Debug token provided by [uploading `pihole -d` log](https://discourse.pi-hole.net/t/the-pihole-command-with-examples/738#debug):**

`{Alphanumeric token}`

**Troubleshooting undertaken, and/or other relevant information:**

I tryed chrome and firefox both with the same reaction.
`{Steps of what you have done to fix this}`

**Comment by PromoFaux** [comment]
Hi there,

Thanks for the report. I must say this is not something I had ever really thought about before, but it is certainly something that we can look into improving.

**Comment by mhussaincov** [comment]
hey there, thanks for being so friendly! :)
would one way of sorting this issue be if a text only version could be
gennerated?
e.g a link that says text only version with say the output of the graphs?
a related issue sorry if i'm out of line in divergeing but the list of
domains via the  Query Log, it is difficult to navigate with orca,
when you click on the link that says query log,
it says showing 100 of 100 queries as an exammple,
when you press b for button it is a shortcut for orca the only thing i
hear is blacklist push button not blacklist google.com push button,
so I am not sure what I am going to blacklist,
navigateing via the arrow keys does not work in this what I believe to
be a table??
sorry for divergeing!
it is linked to the web-ui accessibility after all,
thanks for your assistence! :)
Majid Hussain

On 03/03/2020, Adam Warner <notifications@github.com> wrote:
> Hi there,
>
> Thanks for the report. I must say this is not something I had ever really
> thought about before, but it is certainly something that we can look into
> improving.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pi-hole/AdminLTE/issues/1174#issuecomment-594199724


-- 
kind regards,
Majid Hussain

**Comment by dschaper** [comment]
Our accessibility is not where we'd like it to be. Colors and alternative displays need some work. I agree with Promo that we can look at this. Thank you for reporting it.

**Comment by mhussaincov** [comment]
hey there,
I hope I did not come off as 1, rude or 2, abrasive, incase I did it
was not ment as such.
I am not criticising the project in any way shape or form.
you are the best thing ever!
Majid Hussain

On 03/03/2020, Dan Schaper <notifications@github.com> wrote:
> Our accessibility is not where we'd like it to be. Colors and alternative
> displays need some work. I agree with Promo that we can look at this. Thank
> you for reporting it.
>
> --
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pi-hole/AdminLTE/issues/1174#issuecomment-594203454


-- 
kind regards,
Majid Hussain

**Comment by dschaper** [comment]
You didn't offend me at all, I'm honestly thankful for your bringing this to our attention.

**Comment by PromoFaux** [comment]
Not at all! It's a very valid criticism that we really should take on board, it just never occurred to me before that someone hard-of-sight would be using an ad blocker :)

Presumably adverts get in the way of/ are read out by screen readers?

**Comment by mhussaincov** [comment]
that and popups,
lets not forget the bandwith saveings that are had :)
:)


On 03/03/2020, mhussaincov <notifications@github.com> wrote:
> hey there,
> I hope I did not come off as 1, rude or 2, abrasive, incase I did it
> was not ment as such.
> I am not criticising the project in any way shape or form.
> you are the best thing ever!
> Majid Hussain
>
> On 03/03/2020, Dan Schaper <notifications@github.com> wrote:
>> Our accessibility is not where we'd like it to be. Colors and alternative
>> displays need some work. I agree with Promo that we can look at this.
>> Thank
>> you for reporting it.
>>
>> --
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub:
>> https://github.com/pi-hole/AdminLTE/issues/1174#issuecomment-594203454
>
>
> --
> kind regards,
> Majid Hussain
>
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/pi-hole/AdminLTE/issues/1174#issuecomment-594204257


-- 
kind regards,
Majid Hussain

**Comment by yubiuser** [comment]
Adding `Bug` to this so that it's not going to be stale again. I think accessible issues should not auto-closed, but needs active closing by affected people after an issue is solved.

**Comment by cliffmac19** [comment]
Hi! I'd like to work on this issue. I plan to fix the bug . Could you please assign this issue to me?"
